# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/transfig/multi-line-string-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- transfig.3.2.4/fig2dev/dev/gensvg.c	2002-12-18 23:38:18.000000000 +0100
+++ transfig.3.2.4-jsaw/fig2dev/dev/gensvg.c	2004-03-30 12:40:30.000000000 +0200
@@ -692,16 +692,16 @@
     if (t->angle != 0) {
 	fprintf (tfp, "<g transform=\"translate(%d,%d) rotate(%d)\" >\n",
 		 (int) (t->base_x * mag), (int) (t->base_y * mag), degrees (t->angle));
-	fprintf (tfp, "<text x=\"0\" y=\"0\" fill=\"#%6.6x\"  font-family=\"%s\" 
-		 font-style=\"%s\" font-weight=\"%s\" font-size=\"%d\" text-anchor=\"%s\" >\n",
+	fprintf (tfp, "<text x=\"0\" y=\"0\" fill=\"#%6.6x\"  font-family=\"%s\" "
+		 "font-style=\"%s\" font-weight=\"%s\" font-size=\"%d\" text-anchor=\"%s\" >\n",
 		 rgbColorVal (t->color), family[(int) ceil ((t->font + 1) / 4)],
 		 (t->font % 2 == 0 ? "normal" : "italic"),
 		 (t->font % 4 < 2 ? "normal" : "bold"), (int) (ceil (t->size * 12 * mag)),
 		 anchor[t->type]);
     }
     else
-	fprintf (tfp, "<text x=\"%d\" y=\"%d\" fill=\"#%6.6x\"  font-family=\"%s\" 
-		 font-style=\"%s\" font-weight=\"%s\" font-size=\"%d\" text-anchor=\"%s\" >\n",
+	fprintf (tfp, "<text x=\"%d\" y=\"%d\" fill=\"#%6.6x\"  font-family=\"%s\" "
+		 "font-style=\"%s\" font-weight=\"%s\" font-size=\"%d\" text-anchor=\"%s\" >\n",
 		 (int) (t->base_x * mag), (int) (t->base_y * mag), rgbColorVal (t->color),
 		 family[(int) ceil ((t->font + 1) / 4)],
 		 (t->font % 2 == 0 ? "normal" : "italic"),
